types/vtex.js/index.d.ts

```diff
@@ -0,0 +1,61 @@
+import {SkuJson} from './skuJson'
```

Comment:
IMO this is not a module at all. This is a distribution (through NPM) of a bunch of IFFE modules, so plain JS global scripts, decoupled into bunch of separate JS modules.
It cannot be imported as a module, only via global or script import directly.
Please review handbook, e.g.:
https://www.typescriptlang.org/docs/handbook/declaration-files/library-structures.html#global-libraries